app/application.py

```diff
@@ -143,11 +143,17 @@ def login():
             app.logger.debug(" Values are: ")
             for key in token:
                 logger.debug("{} Value is: {}".format(key, token[key]))
+
         except MissingTokenError as e:
             logger.warning("Missing token error, error is: {}".format(e))
             logger.warning("Failed validation")
             return render_template('sign-in.html', _theme='default', form=form, data={"error": {"type": "failed"}})
 
+        except Exception as e:
+            # logger.error("Error logging in: {}", str(e))
+            print("Error logging in: {}", str(e))
```

Comment:
Oops I didn't mean to leave that print statement in. No, it doesn't seem to be working for me for some reason...?